gulpfile.js

```diff
@@ -87,9 +87,9 @@ gulp.task('javascript', function () {
   if (!process.env.DS_ENV) {
     if (!process.env.TRAVIS_BRANCH
```

Comment:
I actually like this behavior better for my use cases, but unilaterally making the change was an accident.